app/context/extra_keyboard/index.tsx

```diff
@@ -183,9 +183,6 @@ const ExtraKeyboardComponent = () => {
 
     const animatedStyle = useAnimatedStyle(() => {
         let height = keyb.height.value + offset;
-        if (keyb.height.value < 70) {
```

Comment:
This was suppose to make it work with hw keyboard, why does this "fixes" it? What is the result on a phone?

Reply:
Didn't check on a phone. The result on iPad is that when we focus on the input, the on-screen keyboard helper that iOS adds push up the input, so it doesn't overlap.

Reply:
Do check on a phone flet's check on the phone, the easiest way is to use your computer keyboard with the simulator

Reply:
On phone simulator with HW keyboard I don't see any difference.

Reply:
Ok